aiida/engine/processes/calcjobs/tasks.py

```diff
@@ -63,8 +61,8 @@ def task_upload_job(process, transport_queue, cancellable):
         logger.warning(f'CalcJob<{node.pk}> already marked as SUBMITTING, skipping task_update_job')
         raise Return
 
-    initial_interval = TRANSPORT_TASK_RETRY_INITIAL_INTERVAL
-    max_attempts = TRANSPORT_TASK_MAXIMUM_ATTEMTPS
+    initial_interval = get_config().get_option('transport.task_retry_initial_interval')
```

Comment:
a nitpick would be that you could make the actual option strings global variables, e.g.

```suggestion
    initial_interval = get_config().get_option(RETRY_INTERVAL_KEY)
```

Reply:
👍 I've named the global variables `RETRY_INTERVAL_OPTION` and `MAX_ATTEMPTS_OPTION`, since each option also has a `'key'` key, and this is not the one you pass to `get_option()`.